src/renderer/screens/exchange/index.js

```diff
@@ -18,17 +19,19 @@ const Container: ThemedComponent<{ selectable: boolean, pb: number }> = styled(B
 const tabs = [
   {
     title: "exchange.buy.tab",
-    component: <Buy />,
+    component: Buy,
   },
   {
     title: "exchange.history.tab",
-    component: <History />,
+    component: History,
   },
 ];
 
 const Exchange = () => {
   const [activeTabIndex, setActiveTabIndex] = useState(0);
+  const location = useLocation();
   const { t } = useTranslation();
+  const Component = useMemo(() => tabs[activeTabIndex].component, [activeTabIndex]);
```

Comment:
this do not need to be a memo, there is nothing to memorize, we can directly do `Component = tabs[activeTabIndex].component`

Reply:
Makes sense, addressed